ccdaservice/oe-blue-button-generate/lib/entryLevel/functionalStatusEntryLevel.js

```diff
@@ -21,7 +21,7 @@ exports.mentalStatusObservation = {
             attributes: {
                 code: "373930000",
                 codeSystem: "2.16.840.1.113883.6.96",
-                codeSystemName: "SNOMED-CT",
+                codeSystemName: "SNOMED CT",
```

Comment:
hah, you need both spellings of SNOMED w and w/o the dash? and then this entry only wants no dash?